service/src/main/java/org/cbioportal/genome_nexus/service/internal/CuriousCasesServiceImpl.java

```diff
@@ -0,0 +1,96 @@
+package org.cbioportal.genome_nexus.service.internal;
+
+import org.cbioportal.genome_nexus.model.CuriousCasesComment;
+import org.cbioportal.genome_nexus.service.CuriousCasesService;
+import org.cbioportal.genome_nexus.service.exception.CuriousCasesCommentNotFoundException;
+import org.cbioportal.genome_nexus.service.exception.CuriousCasesCommentWebServiceException;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.ResourceAccessException;
+
+import java.util.*;
+
+/**
+ * @author Selcuk Onur Sumer
+ */
+@Service
+public class CuriousCasesServiceImpl implements CuriousCasesService
+{
+    @Override
+    public CuriousCasesComment getCuriousCasesComment(String region) throws CuriousCasesCommentNotFoundException, CuriousCasesCommentWebServiceException
+    {
+        CuriousCasesComment curiousCasesComment = null;
+
+        try {
+            curiousCasesComment = generateCuriousCasesComment(region);
+            if (curiousCasesComment != null) {
+                curiousCasesComment.setRegion(region);
+                return curiousCasesComment;
+            } else {
+                throw new CuriousCasesCommentNotFoundException(region);
+            }
+        }
+        catch (HttpClientErrorException e) {
+            // in case of web service error, throw an exception to indicate that there is a problem with the service.
+            throw new CuriousCasesCommentWebServiceException(region, e.getResponseBodyAsString(), e.getStatusCode());
+        }
+        catch (ResourceAccessException e) {
+            throw new CuriousCasesCommentWebServiceException(region, e.getMessage());
+        }
+    }
+
+    private CuriousCasesComment generateCuriousCasesComment(String region) {
+        CuriousCasesComment curiousCasesComment = null;
+        if (region != null) {
+            if (region.contains(":")) {
+                String[] splited = region.split(":");
+                String chr = splited[0];
+                String range = splited[1];
+                if (range.contains("-")) {
+                    String[] splitedRange = range.split("-");
+                    int start = Integer.parseInt(splitedRange[0]);
+                    int end = Integer.parseInt(splitedRange[1]);
+                    String comment = null;
+                    String pubmedIds = null;
+
+                    if (chr.equals("4") && start >= 55595380 && end <= 55595381) {
```

Comment:
this should be any deletion event from `55593576` to `55593580`, see https://bit.ly/3ewk2hq